sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeConstants.java

```diff
@@ -281,6 +285,38 @@
      */
     static final String PREFERENCE_KEY_TESTER_APP_UPDATE_SETUP_FAILED_MESSAGE_KEY = PREFERENCE_PREFIX + "tester_app_update_setup_failed_message";
 
+    /**
+     * Path where crash logs and temporary files are stored.
+     */
+    public static File DOWNLOAD_FILES_PATH = null;
```

Comment:
We should avoid static constant initialized later, where is this used? Can we switch to reading the context?